.github/workflows/tag-and-build.yml

```diff
@@ -46,18 +86,64 @@ jobs:
         password: ${{ secrets.QUAY_TOKEN }}
         registry: quay.io
 
-    - name: Image Build
+    - name: Image Build and Push
       run: |
         make build
-        make bundle
-        make image-build -e IMG=quay.io/project-codeflare/codeflare-operator:${SOURCE_TAG}
-        podman tag quay.io/project-codeflare/codeflare-operator:${SOURCE_TAG} quay.io/project-codeflare/codeflare-operator:latest
+        make image-build -e IMG=quay.io/${{ github.event.inputs.quay-organization }}/codeflare-operator:${{ github.event.inputs.version }}
+        make image-push -e IMG=quay.io/${{ github.event.inputs.quay-organization }}/codeflare-operator:${{ github.event.inputs.version }}
+
+    - name: Image Push as latest tag
+      if: ${{ inputs.is-latest }}
+      run: |
+        podman tag quay.io/${{ github.event.inputs.quay-organization }}/codeflare-operator:${{ github.event.inputs.version }} quay.io/${{ github.event.inputs.quay-organization }}/codeflare-operator:latest
+        make image-push -e IMG=quay.io/${{ github.event.inputs.quay-organization }}/codeflare-operator:latest
+
+    - name: Build bundle and create PR in OpenShift community operators repository
+      run: |
+        git config --global user.email "codeflare-ci@redhat.com"
```

Comment:
If we use the codeflare-bot github app, is this step necessary?

Reply:
This step is needed as it copies generated bundle and commit it back into repository.
DO we have codeflare-bot github app already somewhere? If so, what does it do?

Reply:
Ah, sorry, I should have said "command". If we have a github bot (the token is available via `CODEFLARE_BOT_TOKEN`), won't the commit just be made as that bot?

Reply:
No, the commit is done on git level. GitHub bot can be used to open a PR, but the commit itself is done using git credentials.